### PR TITLE
[Component] Article Summary

### PR DESCRIPTION
**Pull request by foliveira**
As per #8 last comments

### CODE REVIEW

File: packages/article-summary/article-summary.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+
+const styles = StyleSheet.create({
```
Comment (craigbilner) [inline packages/article-summary/article-summary.js]: for me at least the font sizes are being overridden in the web storybook so it's hard to visually check/demo

Comment (craigbilner) [inline packages/article-summary/article-summary.js]: On native there's no padding which seems odd?

File: packages/article-summary/article-summary.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import "react-native";
+import React from "react";
+import ArticleSummary from "./article-summary";
+import renderer from "react-test-renderer";
+
+it("renders correctly", () => {
+  const tree = renderer.create(<ArticleSummary />).toJSON();
+
+  expect(tree).toBeTruthy();
```
Comment (craigbilner) [inline packages/article-summary/article-summary.test.js]: Can we add a snapshot test?

File: packages/article-summary/article-summary.js (hunk 1 above)
Comment (foliveira) [inline packages/article-summary/article-summary.js]: We need to find a way to inject that into the storybook as a global style

Comment (foliveira) [inline packages/article-summary/article-summary.js]: `storybook-native` doesn't seem to support head scripts injection as `storybook` does, so unless a way around this is found, for the mean time default fonts will appear.

**Pull request merged**